renovate.json

```diff
@@ -28,5 +28,6 @@
   },
   "ignoreDeps": [
     "com.google.protobuf:protoc"
-  ]
+  ],
+  "semanticCommits" : "enabled"
```

Comment:
I'm not sure if semanticCommits = conventional commits, but at least they are very close

Reply:
yeah, it looks like they append "chore" to their commits, which is what we'd want